core/src/main/scala/io/finch/request/package.scala

```diff
@@ -309,7 +358,9 @@ package object request extends LowPriorityImplicits {
    *
    * @param message the message
    */
-  class RequestReaderError(val message: String) extends Exception(message)
+  class RequestReaderError(val message: String, cause: Throwable) extends Exception(message, cause) {
+    def this(message: String) = this(message, null)
```

Comment:
Yeah. Let's keep it with null. I can add a note in the upcoming commits so you probably don't need to update it.